.github/actions/setup-java-gradle/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Setup Java and Gradle
+description: Setup java and gradle with cache for jobs
+inputs:
+  enable-kvm:
+    description: 'Whether to enable KVM'
+    required: false
+    default: true
```

Comment:
KVM is only useful when running the emulator, and only works on Linux.

Maybe it could be `false` by default. And just flip to `true` only for emulator tests.